<launch>

	<arg name="quad" default="SQ01s"/>
	<arg name="goal_topic" default="goal"/>


	<node ns="$(arg quad)" name="faster" pkg="faster" type="faster_node" output="screen" clear_params="true"> 
    <rosparam file="$(find faster)/param/faster.yaml" command="load"/>
    <rosparam file="$(find global_mapper_ros)/cfg/global_mapper.yaml" command="load" ns="mapper"/>

        <!-- Subscriptions -->
		<remap from="~state" to="state"/>
		<remap from="~term_goal" to="/move_base_simple/goal" />
		<remap from="~occup_grid" to="global_mapper_ros/occupancy_grid"/> <!--PointCloud of the occupancy grid-->
		<remap from="~unknown_grid" to="global_mapper_ros/unknown_grid"/> <!--PointCloud of the unknown map-->
		<remap from="~pcloud" to="camera/cloud"/> <!--HARDWARE:    camera/depth/color/points, SIMULATION:  camera/cloud -->
		<!-- <remap from="~odom" to="odometry/local_filtered"/> -->

		<!-- Publications -->
		<remap from="~traj" to="traj"/>
		<remap from="~goal" to="$(arg goal_topic)" />
		<remap from="~setpoint" to="setpoint" />
        <remap from="~log_topic" to="log_topic" />

		<!-- <remap from="~goal_jackal" to="jackal_velocity_controller/cmd_vel" /> -->
		<!-- <remap from="~goal_jackal" to="autonomous_cmd_vel" /> -->		

	</node>

</launch>
